src/App.tsx

```diff
@@ -1,24 +1,156 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { addTodos, deleteTodos, getTodos } from './api/todos';
+import { Header } from './Components/Header';
+import { TodoList } from './Components/Todolist';
+import { Footer } from './Components/Footer';
+import { Notification } from './Components/Notification';
 
-const USER_ID = 0;
+const USER_ID = 6342;
+
+const temp = {
+  id: 0,
+  userId: 0,
+  title: '',
+  completed: false,
+};
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [selectedStatus, setSelectedStatus] = useState('all');
```

Comment:
you can create enum `Filters` and use it everywhere to avoid issues
```suggestion
  const [selectedStatus, setSelectedStatus] = useState(Filters.all);
```